### ISSUE TITLE
RazorpayUtils not defined

### ISSUE THREAD

**Issue by mayankamencherla** [razorpay/razorpay-magento#47]
Class is not defined for some javascript installations. Need to manually include prototype in checkout.phtml.

This worked in the merchant's browser, need to make this more generic and clean.
```
<script type="text/javascript" src="https://checkout.razorpay.com/v1/checkout.js"></script>
<script src="https://ajax.googleapis.com/ajax/libs/prototype/1.7.2.0/prototype.js"></script>
<script type="text/javascript" src="../../../../../../../js/razorpay/razorpay-utils.js"></script>
```

^ needs to be added to `app/design/frontend/base/default/template/razorpay/checkout.phtml` file.

**Comment by captn3m0** [comment]
Did we figure out why the layout was not working?

**Comment by mayankamencherla** [comment]
@captn3m0 Custom theme. Have to dig deeper for this issue. I'll plan this for our next release. Probably next week.

**Comment by captn3m0** [comment]
How do we ensure that we are not including it twice?

**Comment by mayankamencherla** [comment]
Custom Layout - debugging using merchant help.

**Comment by mayankamencherla** [comment]
@captn3m0 We are including it twice. Will figure out a fix for the future.

**Comment by mayankamencherla** [comment]
Analysis: 

- During the time merchant reported this issue we were using prototype, fixed here: https://github.com/razorpay/razorpay-magento/pull/50
- RazorpayUtils was not defined because merchant's magento set up didn't have prototype installed. We saw this issue later and we fixed it by moving the js file to a regular function
- razorpay-utils.js and checkout.js was incorrectly added to the merchant's magento set up, and thereby increasing load time. 
- The current code should work, as I debugged thoroughly and noticed that the layout was being loaded correctly, and it was loading checkout.js and razorpay-utils.js into the head, as defined in razorpay.xml

Confirmed analysis with @pronav sir.

Closing issue as current magento version should have this fixed. Will re-open if issue re-appears.
